include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl.h

```diff
@@ -257,6 +260,32 @@ __parallel_for(_ExecutionPolicy&& __exec, _Fp __brick, _Index __count, _Ranges&&
                                                   ::std::forward<_Ranges>(__rngs)...);
 }
 
+template <typename _ExecutionPolicy, typename _Fp, typename _Index, typename... _Ranges>
+auto
+__parallel_for(oneapi::dpl::__internal::__device_backend, _ExecutionPolicy&& __exec, _Fp __brick, _Index __count, _Ranges&&... __rngs)
```

Comment:
parallel_for should work through the submitter on the new code base. Also it doesn't need kernel name, neither ``__kernel_name_generator``.

What should be done is:
- Please add overload for `__parallel_for` similar to what we have on line 253. It should take the `__device_backend_tag` as the first argument
- Please add overload for `operator()` of the `__parallel_for_submitter` similar to what we have on line 231. It should take the `__device_backend_tag` as the first argument

Reply:
Done